crossing-repair.lic

```diff
@@ -22,7 +22,19 @@ class CrossingRepair
       end
     end
 
-    deposit_coins(@keep_copper, settings) unless settings.sell_loot_skip_bank
+    deposit_coins(@settings.sell_loot_money_on_hand) unless @settings.sell_loot_skip_bank
+  end
+  
+  def deposit_coins(keep_money_by_currency)
```

Comment:
❓ Is this being used instead of the function in `common-money`?

Reply:
It is.  This was copied nearly verbatim from the sell-loot PR.  It should probably be moved to common-money under a different name, but I was in the zone adding town args.  I could cancel this PR, combine it with 2731, and move this method if it's easier, but was thinking they should be separate since they're distinct functionality changes.

Thoughts?